alert/blueprints/application/routes.py

```diff
@@ -11,9 +13,18 @@
 def home():
     form = EditGrammarForm()
     if form.validate_on_submit():
-        graph_script.graph_of_grammar(form.grammar_editor.data)
+        filename = random.randint(1, 10000000)
```

Comment:
You should use the `uuid.uuid4()` function from the `uuid` to reduce risk of collisions. Not a big deal given the bound but with uuids the risk of collision over time is almost 0, whereas using random will lead collisions.